sjb/config/test_cases/test_branch_origin_extended_conformance_install_containerized.yml

```diff
@@ -88,10 +93,11 @@ extensions:
     - type: "script"
       title: "ensure built version of origin is installed"
       timeout: 600
-      repository: "origin"
+      repository: "aos-cd-jobs"
       script: |-
-        origin_package="$( source hack/lib/init.sh; os::build::rpm::format_nvra )"
-        rpm -V "${origin_package}"
```

Comment:
Well, we need the RPMs to build the images but other than I don't think we have any other use for them here.